modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/DocumentResourceTest.java

```diff
@@ -116,6 +120,8 @@ public void testGetDocument() throws Exception {
 			testGroup.getGroupId(), randomDocument(), getMultipartFiles());
 
 		Assert.assertTrue(Validator.isNotNull(document1.getContentUrl()));
+		Assert.assertTrue(Validator.isNotNull(document1.getDateExpired()));
+		Assert.assertTrue(Validator.isNotNull(document1.getDatePublished()));
```

Comment:
@manuelRives, the tests are failing for publishedDate. Based on the changes that I see in the code, shouldn't this be displayed date and not published date?

I would change this and send, but I'm unsure

Reply:
Thank's for taking a look @jkappler!

In theory, getDatePublished should be the correct method. Display date is the internal attribute used in dlFileEntry, but in the Document API we used datePublished in order to be consistent with Journal.

https://github.com/liferay-content-management/liferay-portal/pull/5581/commits/1cbe9a9bd583896da8a8a8f7a4cebb0b0f837663#diff-01ecd6e79dd67f14fdad55c2fb9bf996746fbd5574bd61fbce5f9c12f576a301R543-R558

Also it is strange that this test is failing in our relevant tests but was no failing in the headless tests. I'll check it out further and let you know what I find.

Reply:
@manuelRives, found the issue. FF was set for "LPS" instead of "LPD" in the test. Sent for you